bond_app/authentication.py

```diff
@@ -48,12 +48,6 @@ def __init__(self, accepted_audience_prefixes, accepted_email_suffixes, max_toke
         self.max_token_life = max_token_life
 
 
-def default_config():
-    return AuthenticationConfig(os.environ['BOND_ACCEPTED_AUDIENCE_PREFIXES'].split(),
```

Comment:
let's remove the os import in this file since I don't think it was being used other than here

Reply:
done